packages/blade/src/components/Accordion/Accordion.stories.tsx

```diff
@@ -114,6 +115,21 @@ const AccordionWithIconsTemplate: ComponentStory<typeof AccordionComponent> = ({
 
 export const BasicExample = AccordionTemplate.bind({});
 
+export const WithShowNumberPrefix = AccordionTemplate.bind({});
+
+WithShowNumberPrefix.args = {
+  showNumberPrefix: true,
+};
+
+WithShowNumberPrefix.parameters = {
+  docs: {
+    description: {
+      story:
+        'Use the `showNumberPrefix` prop to automatically add numeric indexes. **Note:** this should not be used with `icon` prop in `AccordionItem`',
```

Comment:
Are we putting any runtime checks for this?

Reply:
Yup